apps/guardian-ui/src/utils/index.ts

```diff
@@ -24,7 +24,8 @@ export const generatePassword = () => {
   };
 
   const passwordLength = 16;
-  const charSet = 'abcdefghjkmnpqrstuvwxyz0123456789';
+  const charSet =
+    'abcdefghjkmnpqrstuvwxyzABCDEFGHJKLMNPQRSTUVWXYZ0123456789!@#$%^&*()';
```

Comment:
@Kodylow unrelated to this pr - I'm not an expert on this stuff, but the random generation code feels a little sketch. Might want to double check with @elsirion on the password generation strategy

Reply:
he okayed it last time, was 80 bit entropy when optional but since we're requiring now we should do the increased charset